pandas/core/frame.py

```diff
@@ -697,7 +697,7 @@ def __init__(
                     "is deprecated and will raise in a future version. "
                     "Use public APIs instead.",
                     DeprecationWarning,
-                    stacklevel=find_stack_level(),
+                    stacklevel=1,  # bump to 2 once pyarrow is released with fix
```

Comment:
Nit: Could you reference the pyarrow version that would have this fix?

Reply:
That's not yet fully sure. At least 15.0.0 I assume, but potentially a 14.0.x release as well. But will add 15.0 then.